atomicapp/cli/main.py

```diff
@@ -365,6 +375,21 @@ def create_parser(self):
             help='The name of a container image containing an Atomic App.')
         gena_subparser.set_defaults(func=cli_genanswers)
 
+        # === "init" SUBPARSER ===
+        init_subparser = toplevel_subparsers.add_parser(
+            "init", parents=[globals_parser])
```

Comment:
Otherwise, https://github.com/rtnpro/atomicapp/blob/atomicapp_init/atomicapp/cli/main.py#L440 breaks.
```
Traceback (most recent call last):
  File "/home/rtnpro/.virtualenvs/atomicapp/bin/atomicapp", line 9, in <module>
    load_entry_point('atomicapp==0.4.5', 'console_scripts', 'atomicapp')()
  File "/home/rtnpro/workspace/projectatomic/atomicapp/atomicapp/cli/main.py", line 501, in main
    cli.run()
  File "/home/rtnpro/workspace/projectatomic/atomicapp/atomicapp/cli/main.py", line 440, in run
    if args.mode:
AttributeError: 'Namespace' object has no attribute 'mode'
```

That's why I had suggested to clean up the CLI in another PR.